tests/console/consoletest_setup.pm

```diff
@@ -68,7 +68,7 @@ sub run() {
     }
 
     # init
-    select_console 'root-console';
+    check_console_font;
```

Comment:
* local reproduction with SKIPTO takes long because you don't have an intermediate step
* if it incompletes within the module all is lost